pandas/core/indexes/base.py

```diff
@@ -4677,7 +4682,8 @@ def values(self) -> ArrayLike:
         """
         return self._data
 
-    @cache_readonly
+    # error: Decorated property not supported
+    @cache_readonly  # type: ignore[misc]
```

Comment:
IIRC there is a mypy issue for this. can you add a link to that issue here as a comment.

Reply:
linked the issue